Overview.bs

```diff
@@ -826,6 +826,9 @@ The algorithm:
     *  If the computed <var>mapped entry index</var> is larger than [=Format 1 Patch Map/maxEntryIndex=] this table is invalid,
         return an error.
 
+    *  If <var>mapped entry index</var> matches an entry index present in the [=Glyph Map=] this table is invalid,
```

Comment:
So, this is the equivalent to having the bin numbers for the feature patches be higher than those for the "base" glyph bins? Am I understanding this correctly?

Reply:
That's right, I'm trying to cleanly separate the inputs and outputs of the feature mappings:
- input entry indices must come from the set of entries specified by glyph map.
- output entry indices must use any entry indices present in the glyph map.

I'm consider for simplicity that we may want to just use the approach you mentioned of requiring the feature output indices to be larger than the highest glyph map index (and the input indices to be less than).

Reply:
I was just confirming I understood the role of this change -- we can make things more specific later if we need to.